packages/strapi-parts/src/Link/Link.stories.mdx

```diff
@@ -10,11 +11,24 @@ import { Stack } from '../Stack';
 
 # Link
 
-This is the doc of the `Link` component
+Links are used to send users to another page or an anchor. They are used for external navigation. 
 
-## Link
+**Best practices**
 
-Description...
+- Use TextButton component if the action is related to internal navigation.
```

Comment:
Nope, a button is not a link

Reply:
You're right. Maybe I could rework the sentence by saying something like "Use TextButton component if an action needs to be triggered within the admin panel"?

Reply:
Yes, you can take the example of opening a modal I think 😊